<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreatView</class>
 <widget class="QWidget" name="TreatView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>4</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>20</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>20</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>65</y>
     <width>250</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 病情描述：</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="description">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>240</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPlainTextEdit" name="diagnose">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>240</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 诊断：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPlainTextEdit" name="advice">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>240</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 建议：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPlainTextEdit" name="prescription">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>240</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 处方：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>39</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>病因：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="disease">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>39</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>38</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>39</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>39</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>患者姓名：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
